docs: add more details for doc and change examples to must have a view (#238)
cmudig/draco2: docs/facts/view.md

# Views

> A view can group marks and scales together. You need to have a view to generate visualization.

Reviewer: ```suggestion
A view can group marks and scales together. You need to define a view before you add any marks.
```
